index.php

```diff
@@ -55,54 +64,72 @@
         return $response;
     }
 
+    $versions = listVersions();
+    $latestVersion = $versions[array_key_first($versions)];
+
     return $this->view->render($response, 'embedded.html.twig', [
-        'layers' => listLayers($selectedRegion),
+        'layers' => listLayers($latestVersion, $selectedRegion),
         'regions' => $regions,
         'selectedRegion' => $selectedRegion,
     ]);
 });
 
 $app->run();
 
-function listLayers(string $selectedRegion): array
+function listLayers(string $version, string $region): array
 {
-    $lambda = new \Aws\Lambda\LambdaClient([
-        'version' => 'latest',
-        'region' => $selectedRegion,
-    ]);
+    $client = new Client();
+    $url = 'https://raw.githubusercontent.com/brefphp/bref/' . $version . '/layers.json';
 
-    $layerNames = [
-        'php-74',
-        'php-74-fpm',
-        'php-73',
-        'php-73-fpm',
-        'php-72',
-        'php-72-fpm',
-        'console',
-    ];
-
-    // Run the API calls in parallel (thanks to async)
-    $promises = array_combine($layerNames, array_map(function (string $layerName) use ($lambda, $selectedRegion) {
-        return $lambda->listLayerVersionsAsync([
-            'LayerName' => "arn:aws:lambda:$selectedRegion:209497400698:layer:$layerName",
-            'MaxItems' => 1,
-        ]);
-    }, $layerNames));
-
-    // Wait on all of the requests to complete. Throws a ConnectException
-    // if any of the requests fail
-    $results = unwrap($promises);
+    $response = $client->get($url);
+    $json = $response->getBody()->getContents();
+    $data = json_decode($json, true);
 
     $layers = [];
-    foreach ($results as $layerName => $result) {
-        $versions = $result['LayerVersions'];
-        $latestVersion = end($versions);
+    $accountId = '209497400698';
+
+    foreach ($data as $name => $regions) {
+        if (!isset($regions[$region])) {
+            continue;
+        }
+
         $layers[] = [
-            'name' => $layerName,
-            'version' => $latestVersion['Version'],
-            'arn' => $latestVersion['LayerVersionArn'],
+            'name' => $name,
+            'arn' => sprintf('arn:aws:lambda:%s:%s:layer:%s:%s', $region, $accountId, $name, $regions[$region]),
+            'version' => $regions[$region],
         ];
     }
 
     return $layers;
 }
+
+function listVersions(): array
+{
+    $client = new Client();
+    $url = 'https://api.github.com/repos/brefphp/bref/releases';
```

Comment:
I wonder if we risk hitting the API limit 🤔 

I guess we'll see, we can always add a basic cache later I think (e.g. cache in `/tmp` or in apcu).

Reply:
We'll probably hit the API limit quickly (I've even experienced it while working on that PR). I'll add some caching.

Reply:
👍